x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/action/saml/TransportSamlInvalidateSessionActionTests.java

```diff
@@ -376,9 +376,9 @@ public void testInvalidateCorrectTokensFromLogoutRequest() throws Exception {
         Authentication authentication = new Authentication(new User("bob"),
                 new RealmRef("native", NativeRealmSettings.TYPE, "node01"), null);
         final Map<String, Object> metadata = samlRealm.createTokenMetadata(nameId, session);
-        final PlainActionFuture<Tuple<String, String>> future = new PlainActionFuture<>();
+        final PlainActionFuture<TokenService.CreateTokenResult> future = new PlainActionFuture<>();
         tokenService.createOAuth2Tokens(userTokenId, refreshToken, authentication, authentication, metadata, future);
-        return future.actionGet();
+        return new Tuple<>(future.actionGet().getAccessToken(), future.actionGet().getRefreshToken());
```

Comment:
nit: I think we can update those methods in here to return a  CreateTokenResult and we could also test that the authentication object is what we expect

Reply:
Done